runtime/lua/vim/lsp/util.lua

```diff
@@ -1828,13 +1833,15 @@ end)
 ---
 ---@param locations table list of `Location`s or `LocationLink`s
 ---@param offset_encoding string offset_encoding for locations utf-8|utf-16|utf-32
----@returns (table) list of items
+---                              default to first client of buffer
+---@return table list of items
 function M.locations_to_items(locations, offset_encoding)
   if offset_encoding == nil then
     vim.notify_once(
       'locations_to_items must be called with valid offset encoding',
       vim.log.levels.WARN
     )
+    offset_encoding = vim.lsp.get_active_clients({ bufnr = 0 })[1].offset_encoding
   end
```

Comment:
This reassigns the `offset_encoding` parameter but doesn't use the value. What is the purpose of that?